scripts/lib/setup/installers.sh

```diff
@@ -216,20 +216,26 @@ function install_node_via_nvm() {
   cd "$(repo_path)"
 
   if [ ! -e "$nvmrc" ]; then
-    cecho "@b@blue[[+ Installing Node 9 (Node 10 is not supported by Realm)]]"
+    cecho "@b@blue[[+ Installing Node 8 (Node 10 is not supported by Realm and Node 9 doesn't support npm 5.5.1)]]"
 
-    nvm install 9
-    nvm alias status-im 9
+    nvm install 8.9.4
```

Comment:
I use “nvm use 9.3.0”

Reply:
Thanks @mandrigin. The question is whether we should strive to have an environment which works both for building desktop and mobile. If so, desktop is currently [asking](https://status.im/build_status/desktop.html) to install Node 8/npm 5.5.1. @siphiuel does desktop really require Node 8 and npm 5.5.1, or can it work with Node 9?

Reply:
I would say we should have different code paths then for desktop and mobile and a task to try to migrate Desktop to a more modern node version :)

Reply:
also, you can use an `.nvmrc` file to simplify that logic

Reply:
@PombeirP there are issues with Node versions higher than 8.9.4/npm versions higher than 5.5.1 due to hanging `npm install`. Node 10 is not supported as Realm did not support Node 10 until very recently (https://github.com/realm/realm-js/releases/tag/v2.19.0). Most likely upgrading Desktop to higher Node versions will not be necessary as @MaxRis is working on substituting Node.js with Qt's built-in WebEngine.

Reply:
@PombeirP (replying to your comment here: https://github.com/status-im/status-react/pull/6833#issuecomment-441596885) one thing is to add a comment with the explanation on why we are installing that version of node. And, possibly, a warning message to `startdev-<blah>` commands if the version isn't matching.

Reply:
@mandrigin I'd already added a comment to https://github.com/status-im/status-react/pull/6833/files#diff-0bf2011909118cdaef211364576e068dR226, and will add a check to those commands to ensure we have the right npm version, that's a good thing to do to ensure sanity of the build.

Reply:
Done.